### ISSUE TITLE
Convert es5 to es6 code

### ISSUE THREAD

**Issue by zamhaq** [fossasia/susi_chromebot#170]
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Script.js contains es5code
**Expected behavior:**
Replace es5 code to es6 wherever applicable. For example replace ordinary functions with arrow functions. 

**Would you like to work on the issue?**
No.

**Comment by zamhaq** [comment]
Putting this up for code-in

**Comment by ms10398** [comment]
:+1:

**Comment by techpool** [comment]
I would like to take this up.

**Comment by ms10398** [comment]
This is a task put for Google Code-in @techpool

**Comment by arnav-mandal1234** [comment]
Hello Sir,
I am a newbie to opensource.Can I take this task?

**Comment by stealthanthrax** [comment]
@arnav-mandal are you a Gci student?

**Comment by arnav-mandal1234** [comment]
No, I am a GSoC aspirant.It would be mine first task for the open source.If I can take this task it will be helpful for me to make a contribution to harder stuff and projects.

**Comment by stealthanthrax** [comment]
@arnav-mandal1234 ,but this task is reserved for gci students . You can browse other issues.
Also, no need to call us sirs 😄

**Comment by arnav-mandal1234** [comment]
okay, no problem But I could get it I would have helped me a lot!

**Comment by ms10398** [comment]
As no GCI student picked it up
I will work on it.

**Comment by anshumanv** [comment]
@ms10398 collaborating. :smile:

**Comment by rupav** [comment]
@anshumanv @ms10398 can I start working on it? I will use `let` and `const` instead of `var`. Also will replace all functions with fat arrow ones.

**Comment by zamhaq** [comment]
> @anshumanv @ms10398 can I start working on it? I will use `let` and `const` instead of `var`. Also will replace all functions with fat arrow ones.

you can go ahead I guess. It's been a long time since the issue is stale.

**Comment by rupav** [comment]
@ms10398 @zamhaq this can be reopened, as more work is still to be done in it. I am taking up next part of the issue.

**Comment by ammarbinfaisal** [comment]
so.., is webpack going to be used? should I add it?
